clientd3d/d3drender.c

```diff
@@ -7025,9 +7025,9 @@ void D3DRenderObjectsDraw(d3d_render_pool_new *pPool, room_type *room,
 		pChunk->xLat0 = xLat0;
 		pChunk->xLat1 = xLat1;
 
-		pChunk->zBias = ZBIAS_BASE;
-		// Nodes with a bound height adjust are part of other players' upper bodies.
-		if (pRNode->boundingHeightAdjust == 0)
+		pChunk->zBias = (pRNode->obj.flags & OF_PLAYER) || (pRNode->boundingHeightAdjust!=0) ? ZBIAS_BASE : ZBIAS_DEFAULT;
+
+		if (pRNode->obj.flags & OF_GETTABLE)

```

Comment:
Please add a comment explaining what's going on in these two lines.  Without that, this looks like voodoo magic.

Why was the previous comment here about player bodies removed, was it wrong?